### DISCUSSION TITLE
Visual Editor. Center image ?

### DISCUSSION THREAD

**Discussion by Scrik** [requarks/wiki#3391] [Help / Questions]
**Question**

I insert a picture, select it in the center, but it is always on the left edge.

![image](https://user-images.githubusercontent.com/5807097/84798565-c97ca100-b003-11ea-806c-b5a611a7b7e0.png)

**Host Info (please complete the following information):**
OS: Centos 7
Wiki.js version: Stable 2.4.107
Database engine: postgres 11

**Comment by Zyzto** [comment]
having the same issue idk it's seems to be ignored like no one center images or what.
also tables can't be centered

**Comment by Hearbal** [comment]
A similar problem. The image is not centered. Tell me how to update the visual editor to the current version of CKEditor 5. They have new useful functions.

**Comment by riccardospeggiorin-centropaghe** [comment]
same problem here
